fabrics.c

```diff
@@ -110,15 +112,6 @@ static const char *nvmf_config_file	= "Use specified JSON configuration file or
 	OPT_FLAG("data-digest",       'G', &c.data_digest,        nvmf_data_digest), \
 	OPT_FLAG("tls",                 0, &c.tls,                nvmf_tls)	\
 
-struct tr_config {
-	const char *subsysnqn;
-	const char *transport;
-	const char *traddr;
-	const char *host_traddr;
-	const char *host_iface;
-	const char *trsvcid;
-};
-
```

Comment:
The refactoring part should be in a preparation patch.

Reply:
I've broken this change out into a separate commit.